tests/test_gs_chunked_io.py

```diff
@@ -42,14 +43,27 @@ def tearDownModule():
     GS.client._http.close()
 
 class TestGSChunkedIOWriter(unittest.TestCase):
-    WriterClass = gscio.Writer
-
     def setUp(self):
         _suppress_warnings()
 
+    def duration_subtests(self):
```

Comment:
Have you considered `pytest --durations=0`?

Reply:
I'll look into it.

Reply:
This will be a future PR if it looks useful.